theme-dev/README.md

```diff
@@ -111,11 +110,11 @@ If done correctly it should look similar to this in the .JSON
 Only the links listed below are accepted for external resources. Other links will refuse to load.
 This is for security and privacy reasons.
 
-
+Note: cdn links expire as said [here]
```

Comment:
Please fix the markdown here

Reply:
also cdn.discordapp.com and media.discordapp.net links flat out don't work, so that should be stated instead of "expire"

Reply:
Done, hopefully.

Reply:
> also cdn.discordapp.com and media.discordapp.net links flat out don't work, so that should be stated instead of "expire"

added media too.